main.py

```diff
@@ -1,6 +1,13 @@
 """ToDo."""
 from tasks.kyu8 import example
+from tasks.kyu8 import volume_of_a_cuboid
 
+def calculate_volume():
+    """Run calculation pf a volume of a cuboid."""
+    length = input("length: ")
```

Comment:
Please, add check (_try...except_) and rewrite all input like _length = float(input("length: "))_.